packages/pn-commons/src/utils/user.utility.ts

```diff
@@ -43,3 +43,27 @@ export function basicInitialUserData<T extends BasicUser>(yupMatcher: yup.Object
     return noLoggedUserData;
   }
 }
+
+export function adaptedTokenExchangeError(originalError: any, tokenName: string) {
```

Comment:
Come mai ci sono queste due casistiche e perchè si hanno due messaggi diversi?
In generale localizzare tutte le stringhe come fatto da altre parti nella libreria.

Reply:
Ci sono due motivi diversi per cui l'accesso tramite `exchangeToken` viene rifiutato: 
(1) il token è valido ma all'utente mancano grants che sono necessari per accedere all'app alla cui vuole entrare.
(2) il token non è valido.

Questo è descritto nel commentone in inglese, proprio all'inizio, "Conditions to verify". Infatti c'è una terza casistica, cioè non arriva un token da SPID/SelfCare, ma in quel caso non viene chiamata l'API `exchangeToken`.

Reply:
Per quanto riguarda la localizzazione, la posso fare, vi prego confermarmi che la faccio io (per evitare conflitti con eventuali commits fatti degli reviewers)

Reply:
@clombardi procedi tu con la localizzazione